### PR TITLE
Minor restructure

### PR DESCRIPTION
**Pull request by corinnebosley**
No code in this PR; just moving stuff around and getting rid of outdated files.  This is what I've done:
- Removed 'publish' directory
- Created docs directory to contain all MD and HTML files (later we will have to make this properly, but not yet)
- Removed 'users' directory with unused 'user_manager.py' file
- Removed visualise/assets and visualise/works directories, and either moved potentially useful files to docs directory or just deleted them
- Removed empty and unused unit tests for 'render_map.py' and 'render_plot.py'
- Created a new directory to contain loose clean_air modules ('exceptions.py', 'models.py' and 'serialisation.py') called 'base'.  Happy to rename this, just couldn't think of what to call it.

### CODE REVIEW

**Comment by corinnebosley** [comment]
It appears that I totally failed to stash my changes to my previous branch, so they're in here too.  They don't work, obviously, but maybe we could merge anyway in the name of pragmatism so that I can just keep working on them.  

It's your call really.  I can spend a lot of time figuring out how to stash those changes to another branch before we merge this or we could just merge it as is and pretend it never happened.

**Comment by corinnebosley** [comment]
OK, from looking at the tests I can see that for some reason we need the 'publish' directory, so I'll chuck it back in.

File: docs/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Credentials must be provided by
```
Comment (tomw66) [inline docs/README.md:33]: I think we can leave the README at the top level - standard for most projects and tox is complaining it can't find it

**Comment by tomw66** [comment]
I was slightly concerned that the files in `visualise/assets` were those outputted by the `generate_map_based_visualisations` integration tests, but actually these seem to output elsewhere. I think at least 2 of these files also exist in the notebook repository, so we could remove the ones here?

**Comment by tomw66** [comment]
I've re-removed the publish directory - tests seem fine now (had to remove 'publish' from the top-level init file too)

**Comment by tomw66** [comment]
(I'm going to merge this now so I can branch off main)

File: docs/README.md (hunk 1 above)
Comment (corinnebosley) [reply docs/README.md:33]: No probs, I'll change that back now.

**Pull request merged**